apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts

```diff
@@ -32,18 +37,18 @@ export class WizardModalComponent implements OnInit {
   }
 
   public checkRequirementsMet(): void {
-    this.step2Disabled = !this.myForm.get('requiredValue1')?.value;
-    this.step3Disabled = !this.myForm.get('requiredValue2')?.value;
-    this.saveDisabled = this.myForm.get('requiredValue3')?.value;
+    this.step2Disabled = !this.myForm?.get('requiredValue1')?.value;
```

Comment:
You can create `myForm` in the constructor and save yourself that first `?`.
```
public myForm: UntypedFormGroup;

constructor(formBuilder: UntypedFormBuilder) {
  this.myForm = formBuilder.group({...
}

// ...

this.step2Disabled = !this.myForm.get('requiredValue1')?.value;
```

Reply:
changed